src/app.ts

```diff
@@ -332,29 +332,45 @@ function optimizeRow(
   const preferGeneratedAttributes = getConfigSheetValue(
     CONFIG.userSettings.title.preferGeneratedAttributes
   );
+
+  const blocklistedAttributes = getConfigSheetValue(
+    CONFIG.userSettings.title.blocklistedAttributes
+  )
+    .split(SEPARATOR)
+    .filter(Boolean)
+    .map((x: string) => x.trim().toLowerCase())
+    .filter(
+      (x: string, index: number, array: string[]) => array.indexOf(x) === index
```

Comment:
I don't get why there's a `filter` here - aren't those user provided values? Could you provide an example?

Please also adjust the README to explain the new config value.

Reply:
tried to achieve Set semantics for some inexplicable reason - removed